roles/ceph-defaults/tasks/facts.yml

```diff
@@ -239,3 +239,30 @@
     - ceph_current_status['servicemap'] is defined
     - ceph_current_status['servicemap']['services'] is defined
     - ceph_current_status['servicemap']['services']['rgw'] is defined
+
+- name: get default value for osd_pool_default_pg_num
+  shell: |
+   {{ docker_exec_cmd }} ceph --cluster {{ cluster }} daemon mon.{{ hostvars[groups[mon_group_name][0]]['inventory_hostname'] }} config get osd_pool_default_pg_num
```

Comment:
This would only work if a cluster already exists. This command must run from an OSD so that we are sure the cluster is up.

Reply:
Removed: 
failed_when: false
added:
when:
    - not rolling_update
    - ceph_current_status.rc == 0
The next task already has the condition to check for this task rc.